content-repo/extra-docs/articles/XSOAR-Build.md

```diff
@@ -0,0 +1,69 @@
+## Automatic Configuration Setup Process (Alpha)
+### Purpose
+This process encapsulates what you need in order to control your XSOAR machines in an automated manner, while providing the ability to manage your own content, in your artifacts server of choice, with your version control system of choice.
+
+---
+
+### The `xsoar_config.json` File
+The configuration file that defines what will be set up on the machine.<br /> 
+Consists of the following sections:
+1. `custom_packs` - Your own internal packs to be installed through the build process.
+2. `marketplace_packs` - Marketplace packs to be installed on the machine.
+3. `lists` - Lists to be created in the machine.
+4. `jobs` - Jobs to be created in the machine.
+
+[Example file](https://raw.githubusercontent.com/demisto/content/aace565faff531f09a42268b897d629981e69b08/Packs/XSOARbuild/docs-files/xsoar_config.json)
+
+---
+
+### The `Configuration Setup` Incident type
+#### Custom Fields:
+3. `Configuration File Source` - The source of the configuration file.
+3. `Custom Packs Source` - The source of the custom packs.
+1. `Configuration File Path` - The relative path within the repository to the `xsoar_config.json` file. (Optional)
+2. `Branch Name` - The branch from which to fetch the file. Default is the main branch. (Optional)
+
+#### Layout:
+![layout.png](https://raw.githubusercontent.com/demisto/content/aace565faff531f09a42268b897d629981e69b08/Packs/XSOARbuild/docs-files/layout.png)
+
+---
+
+### The `Configuration Setup` Playbook
+This playbook will manage the entire configuration process and needs to run through a `Configuration Setup` incident type.<br />
+It consists of five sections:
+1. Fetching the configuration file and loading it's content to the machine.
+2. Downloading and installing the custom packs.
+3. Installing marketplace packs.
```

Comment:
shouldn't the custom packs be installed after the marketplace packs?

Reply:
It's being done in parallel.